modules/ROOT/pages/app-to-app-requests.adoc

```diff
@@ -0,0 +1,23 @@
+= Access an Application in the Same Cluster
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In an Anypoint Runtime Fabric cluster, every application replica exposes an endpoint inside the cluster. When you enable inbound traffic, that endpoint is exposed externally. Refer to xref:enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric] for additional information about servicing external requests and configuring the load balancer. 
```

Comment:
This seems like an odd way to introduce a completely separate task, especially by sending the audience off to look up that other task. Suggest rewording to actually lead in to servicing internal requests.

Reply:
Also, why "application replica"?